c2corg_ui/templates/outing/view.html

```diff
@@ -0,0 +1,161 @@
+<%!
+from c2corg_ui.templates.utils import get_lang_lists
+import json
+%>
+<%inherit file="../base.html"/>
+<%namespace file="../helpers/common.html" import="show_title"/>
+<%namespace file="helpers/detailed_outings_attributes.html" import="get_outing_associated_maps, get_outing_snow, get_outing_access, get_outing_participants, get_outing_location, get_outing_general, get_outing_heights"/>
+<%namespace file="../helpers/view.html" import="get_document_min_max, get_document_locale_text, show_attr, show_missing_langs_links,
+    show_other_langs_links, show_archive_data, show_route_title, get_route_activities, 
+    show_areas, show_maps, show_float_buttons, get_associated_documents"/>
+<%
+outing_id = outing['document_id']
+other_langs, missing_langs = get_lang_lists(outing, lang)
+geometry4326 = outing['geometry']
+point = json.loads(outing['geometry']['geom'])
```

Comment:
This line is responsible of the crashes I have observed when loading an outing detail view => if the outing has no geometry set (``None``), python crashes saying that ``TypeError: the JSON object must be str, not 'NoneType'``.

My API instance seems outdated since I get no error when using the API demo instance.
In the new DB structure ``outing['geometry']['geom']`` is the default point geomtry and is always available for all outings. @tsauerwein is it correct?

Anyway I don't see where the ``point`` variable is used in the template!?